src/usr/local/www/widgets/widgets/openvpn.widget.php

```diff
@@ -46,7 +46,12 @@ function printPanel($widgetkey) {
 			$got_ovpn_server = true;
 
 		$opstring .= "<div class=\"widget panel panel-default\">";
-		$opstring .=	"<div class=\"panel-heading\"><h2 class=\"panel-title\">" . htmlspecialchars($server['name']) . "</h2></div>";
+		$opstring .= "<div class=\"panel-heading\"><h2 class=\"panel-title\">" .
+			htmlspecialchars($server['name']);
+		if ($server['conns'][0]['common_name'] != '[error]') {
+			$opstring .=  gettext('Client Connections') . ": " . 
+				sizeof($server['conns']) . "</h2></div>";
```

Comment:
- Multiline statements are indented with TABs followed by 4 spaces
- You should remove `</h2></div>` from here and concatenate it outside if statement to get these tags closed when common_name == error